openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -646,8 +650,9 @@ def _create_course(self, course_price, course_run_count=1):
 
         run = CourseRunFactory(key=unicode(course.id), seats=[SeatFactory(price=course_price)])
         course_runs.append(run)
+    entitlements = [EntitlementFactory() for x in range(entitlement_count)]
```

Comment:
If x is greater than 1, this will result in multiple `verified` entitlements being created. Does that create any problems?

Reply:
Yes that would cause an IntegrityError. I will update it.